templates/problem/problem.html

```diff
@@ -304,6 +317,15 @@ <h2 style="display: inline-block">{{ title }}</h2>
         {{ description|markdown(problem.markdown_style, MATH_ENGINE)|reference|str|safe }}
     {% endcache %}
 
+    {% if problem.authorship_name %}        
+        <span class="authorship">
+            <hr>
+            {% trans trimmed %} Authorship: {% endtrans %}          
+            <a href="{{ problem.authorship_uri }}">{{ problem.authorship_name }}</a>
```

Comment:
From the model definition, it seems that `problem.authorship_uri` can be blank. Let's add a check here when the URI is blank to not create a link.

Reply:
Done :)